### DISCUSSION TITLE
Stuck in the application process

### DISCUSSION THREAD

**Discussion by lynaxel** [community/community#143496] [GitHub Education]
### Hi everyone,

In the application page,  I click continue button, browser ask location permission and I click allow. Then the process is stuck, the web page change nothing and don't redirect to another page. I try it on my PC and phone, all devices show same. I have no idea how to continue
![image](https://github.com/user-attachments/assets/74ffa6d9-b3cf-4a44-9004-93b2fce05fc8)

**Comment by davevad93** [comment]
Hi @lynaxel ,

This is usually due to a browser cache issue, try one of the following solutions:

1. Clear the cache.
2. Use incognito mode.
3. Use another browser.
4. Submit the application via phone in incognito mode.

**Reply by lynaxel** [reply]
hello @davevad93 ,
Thanks for your advice, I've tried all these ways before I post this issue, but not work for me.
